src/styles/blocks/section.scss

```diff
@@ -0,0 +1,21 @@
+.main__section {
+  display: flex;
+  width: 1024px;
+  align-items: center;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  box-sizing: border-box;
```

Comment:
I see, that you often use `box-sizing: borderd box` so you can just write once it's, for example: `* { box-sizing: border-box }`

Reply:
this mistake was fixed